composer.json

```diff
@@ -12,9 +12,9 @@
     "behat/mink-extension": "^2.2",
     "behat/mink-goutte-driver": "^1.2",
     "pantheon-systems/pantheon-wordpress-upstream-tests": "dev-master",
-    "wp-coding-standards/wpcs": "^2",
+    "wp-coding-standards/wpcs": "dev-develop as 2.3.1",
     "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
-    "phpunit/phpunit": "^7",
+    "phpunit/phpunit": "^9",
     "phpcompatibility/php-compatibility": "^9.3",
```

Comment:
Note php-compatibility ^9.3 does not have any PHP 8 testing (although default branch does)